src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -215,7 +221,15 @@ const styles = StyleSheet.create({
     fontSize: 14,
     fontStyle: 'normal',
     fontWeight: Platform.OS === 'android' ? '700' : '600',
-    lineHeight: 22,
+    marginBottom: 12
+  },
+  bioTextNull: {
+    color: COLORS.lightgrey,
+    fontFamily: 'Inter',
+    fontSize: 14,
+    fontStyle: 'italic',
+    fontWeight: '400',
+    lineHeight: 20,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new style `bioTextNull` is well defined and used for the default bio message when the `bio` prop is empty. The use of 'italic' fontStyle and lighter fontWeight differentiates it from the actual bio text, which is a good UX practice.
```diff
+ bioTextNull: {
+   color: COLORS.lightgrey,
+   fontFamily: 'Inter',
+   fontSize: 14,
+   fontStyle: 'italic',
+   fontWeight: '400',
+   lineHeight: 20,
+   marginBottom: 12
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->